blackbox/docs/configuration.txt

```diff
@@ -1930,6 +1930,34 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Enterprise Features
+===================
+
+CrateDB Licensing
+-----------------
+
+The CrateDB licensing information.
+
+.. _license_enterprise:
+
+**license.enterprise**
+  | *Default:*  ``false``
+  | *Runtime:*  ``no``
+
+  The ``enterprise`` setting indicates whether the `Enterprise Edition`_ of
```

Comment:
what about: Setting this setting to true enables CrateDB's enterprise features.

so that it sounds like a setting not a read property ... this sentence could probably be used in the config file too